venues/DBLP.org/python/superuser-init.py

```diff
@@ -112,7 +112,7 @@ def overwrite_allowed(groupid):
                 'description': 'Comma separated list of author email addresses, in the same order as above.',
                 'order': 3,
                 'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
-                'required':True
+                'required':False
```

Comment:
this should applies to the 'DBLP.org/-/Add/Revision' invitation reply only, not for the 'DBLP.org/-/paper'. 

BTW DBLP.org/-/paper should be renamed to DBLP.org/-/Submission, @mspector  what do you think?